bluesky/plans.py

```diff
@@ -751,15 +765,17 @@ def inner_product_scan(detectors, num, *args, per_step=None, md=None):
     :func:`bluesky.plans.grid_scan`
     :func:`bluesky.plans.scan_nd`
     """
+    num = args[-1]
```

Comment:
Does this need a `if not len(args) - 1 % 3 == 0: raise` logic?

Reply:
That check happens in `plan_patters.py`. (I know because I tripped it.) Doesn't seem necessary to do it again here.